schemas/groups/sails.json

```diff
@@ -58,6 +58,32 @@
               "units": "m/s",
               "default": 666
             },
+            "reducedState": {
+              "oneOf": [
+                {
+                  "properties": {
+                    "Reefable": {
```

Comment:
@joabakk `Reefable` should be `reefable` as we use `camelCase` everywhere.

If I read this correctly, it comes out something like this:
`reducedState.reefable.reefs = 0`
`reducedState.furlable.furledRatio = 0`

Where the "reefable" and "furlable" seems unnecessary and incorrect English. Also, referring back to your comment on #188, it may be nice to have a user-friendly boolean property "reduced". So the model would look something like:

```javascript
reducedState: {
  reduced: true,
  reefs: 1,
  furledRatio: 0
}
```